src/main/kotlin/no/nav/syfo/infrastructure/kafka/VurderingProducer.kt

```diff
@@ -49,7 +49,7 @@ data class VurderingRecord(
                 personident = vurdering.personident.value,
                 veilederident = vurdering.veilederident,
                 type = vurdering.type,
-                begrunnelse = vurdering.begrunnelse,
+                begrunnelse = vurdering.begrunnelse ?: "",
```

Comment:
Vil vi endre Kafka-skjemaet også til å være nullable for begrunnelse?

Reply:
Kanskje greit å la den være hvis det kommer begrunnelse her etter hvert.